boards/uvify/core/init/rc.board_extras

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# UVify UVF4 specific board extras init
+#------------------------------------------------------------------------------
+# IFO
+if param compare SYS_AUTOSTART 4053
+then
+   # Wifi module : 1st telemetry
+   mavlink start -d /dev/ttyS1 -b 921600 -m onboard
+   # Sub 1-Ghz : 2nd telemetry
+   mavlink start -d /dev/ttyS2 -b 57600 -m onboard
+
```

Comment:
This introduces a conflict with the parameters configuring the telemetry ports such as `SER_TEL1_BAUD`, `MAV_0_CONFIG`, `MAV_0_MODE` .etc Is there a reason that this is hardcoded into this file?

Reply:
That's inherited from our old PX4 maintenance. It's ok to *remove*.

Reply:
We will prepare another PR for this.

Reply:
@limhyon It would be helpful if you add it to this PR